sdk/monitor/monitor-opentelemetry/src/types.ts

```diff
@@ -123,6 +123,7 @@ export enum StatsbeatFeature {
   BROWSER_SDK_LOADER = 4,
   DISTRO = 8,
   LIVE_METRICS = 16,
+  SHIM = 32,
```

Comment:
StatsbeatFeature enum source of truth should be in this package and not being defined in multiple places, is that the case now?

Reply:
We discussed this is new functionality, env variable had been used by distro/exporter for a while but not shim

Reply:
This enum is replicated here: https://github.com/microsoft/ApplicationInsights-node.js/pull/1331/files#diff-c3e565081a23e128a1b1d782dbec7cdaa49647a0baea4f9b7d06f0d85e442fe5. I'll clean that up and expose our this as an internal type.